manifests/charts/ztunnel/templates/daemonset.yaml

```diff
@@ -244,9 +244,6 @@ spec:
           httpGet:
             port: 15020 # TODO: use 15021, but it spams logs
```

Comment:
We should probably make this 15021 (should work for envoy) then switch the ztunnel code to use 15021 as well.

In the meantime I suspect it won't get marked as ready